### PR TITLE
Add theme changer

### PR DESCRIPTION
**Pull request by riceissa**

### CODE REVIEW

**Review by vipulnaik — CHANGES_REQUESTED** [review]
Can we reserve space in the pre-JS code (e.g. a blank line) so that the Change color" just replaces the already reserved space rather than pushing things down?

File: access-portal/index.php [hunk 1]
```diff
@@ -29,6 +31,7 @@
     }
 ); </script>'."\n";
 print "<h3>Donations recorded by Vipul Naik $causeAreaFilterStringHelper</h3>";
+print '<span id="changeThemeMenu" style="display: none;">Change <a href="#" onclick="change_theme_color()">color</a></span>';
```
[the remark this answers is not in the document]
Comment (riceissa) [reply access-portal/index.php:34]: `href="javascript:;"` seems to work https://stackoverflow.com/a/7676563/3422337

**Comment by riceissa** [comment]
I have made the requested changes

**Review by vipulnaik — CHANGES_REQUESTED** [review]
I'm still seeing the cumulative layout shift on Chrome at https://donations.vipulnaik.com/donor.php?donor=Open+Philanthropy (though the other issue is fixed!). I suspect the blank paragraph ends up being ignored by the browser in terms of reserving space. Can you look more into this? Thank you!

**Comment by riceissa** [comment]
I've tried a unicode thin space now instead. It seems to work when I test it locally (though the page loads too fast for me to tell if it's really working).

**Review by vipulnaik — APPROVED** [review]
Thnaks @riceissa -- this works!

**Comment by Rms125** [comment]
👁️👍

**Pull request merged**